rhombus/src/internal/discord.rs

```diff
@@ -448,19 +448,19 @@ pub async fn ai(ctx: Context<'_>) -> std::result::Result<(), DiscordError> {
     let challenge_data = ctx.data().db.get_challenges().await.unwrap();
     let challenge = challenge_data
         .challenges
-        .iter()
+        .values()
         .find(|c| c.id == ticket.challenge_id)
         .unwrap();
     let category = challenge_data
         .categories
-        .iter()
+        .values()
         .find(|c| c.id == challenge.category_id)
         .unwrap();
```

Comment:
Now that these are `BTreeMap`s can you just look them up directly instead of `find`ing them?